src/dao/migrations/20170909162220_create-installation-table.js

```diff
@@ -0,0 +1,22 @@
+const logger = require('~/src/logging').logger(module)
+const Installation = require('~/src/models/github/Installation')
+
+const INSTALLATION_TABLE_NAME = Installation.typeName
+
+exports.up = function (knex, Promise) {
+  logger.info('Attempting to run "up" on "create-installation-table"')
+  logger.info('Attempting to create installation table with table name: ', INSTALLATION_TABLE_NAME)
+
+  return knex
+    .schema
+    .createTableIfNotExists(INSTALLATION_TABLE_NAME, (table) => {
+      // TODO: Add more as more properties are added
+      // to the Installation model
+      table.integer('id').unsigned().primary()
+      table.integer('app_id').notNullable()
+      table.timestamps(true, true) // adds created_at/updated_at columns and default to now
```

Comment:
Not actually sure if we'll ever care about these timestamps, but seemed like it'd be good to know create/updated times.

Reply:
Agreed.